### DISCUSSION TITLE
Enabling private github packages

### DISCUSSION THREAD

**Discussion by jeconneely** [community/community#24554] [Apps, API and Webhooks]
<p>We are trying to create our first private git-hub package, but we get the following error:</p>
<p>npm ERR! code EPRIVATE<br>
npm ERR! This package has been marked as private<br>
npm ERR! Remove the ‘private’ field from the package.json to publish it.</p>
<p>How do we enable private packages? I see from <a href="https://github.com/features/packages" rel="nofollow noopener">https://github.com/features/packages</a> that it says “GitHub Packages is not available for private repos in legacy per-repository plans.”  How do I determine if that applies to me?  We have had a plan allowing private repositories for some time, but I don’t think it is “per-repository”, it’s based on our organization.   Is there something else I need to do to enable private packages?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeconneely">@jeconneely</a>,</p>
<p>Thank you for being here! I’m sharing the reply from our support team as it might help someone in the same situation:</p>
<blockquote>
<p>npm ERR! Remove the ‘private’ field from the package.json to publish it.</p>
</blockquote>
<p>npm has a confusing definition of private packages. They are packages that should never be published <em>anywhere</em> (hence the error).</p>
<p>What we mean by private packages is packages that are associated with a private repository. You will need a PAT with access to that repository and the <code>read:packages</code> scope in order to access them</p>
<blockquote>
<p>How do we enable private repositories?</p>
</blockquote>
<p>You should remove the <code>private</code> field and instead associate the package with a private repository.</p>
<p>See: <a href="https://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages#publishing-multiple-packages-to-the-same-repository" rel="nofollow noopener">https://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages#publishing-multiple-packages-to-the-same-repository</a></p>
